src/mvnw.js

```diff
@@ -174,8 +170,21 @@ function print() {
         }
       }
     }
-    return curr;
+  } catch (err) {
+    if (err.code === 'ENOENT') {
+      console.info(`Directory or file is not found: ${dir}`);
```

Comment:
@trapvpack exception swallowing doesn't look like a solution to me. Instead, let's try to find the root cause of the problem and fix it.

Reply:
@yegor256 After looking through `mvnw.js`, I could only trace a possible problem with calling `start` before closing `spawn`, but I'm not at all sure about this. The questions are: 
 - Do you have any ideas about what we should pay attention to? 
 - Could this be a bug outside of `mvnw.js`?

Reply:
@trapvpack are you sure the tests you created reproduce the problem? Do you see the error when you run them (without your fix)?